alpine/Containerfile

```diff
@@ -93,6 +93,7 @@ RUN set -eux; \
             .. \
             -DCMAKE_INSTALL_PREFIX=/opt/weechat \
             -DENABLE_MAN=ON \
+	          -DENABLE_HEADLESS=ON \
```

Comment:
Please replace the Tab by spaces.

Reply:
You should also enable headless when building slim container (in the else below).